lib/embeds/instagram.js

```diff
@@ -18,7 +18,16 @@ export default {
       : '';
 
     const script = `<script src="${protocol}//platform.instagram.com/en_US/embeds.js"></script>
-      <script>resize()</script>`;
+      <script>
+        const interval = setInterval(function () {
+          const iframe = document.querySelector('iframe.instagram-media-rendered');
+          if (iframe && iframe.height) {
+            resize();
+            clearInterval(interval);
+          }
+        }, 100);
```

Comment:
why does it need 100ms to work properly? is there a more declarative way of doing this? like w/ `document.addEventListener('DOMContentLoaded', fn)` or something?